includes/class-amp-story-post-type.php

```diff
@@ -1362,26 +1362,23 @@ public static function render_block_latest_stories( $attributes ) {
 	}
 
 	/**
-	 * Flushes rewrite rules if it hasn't been done yet after having AMP Stories Post type.
+	 * Add RSS feed link for stories.
+	 *
+	 * @since 1.2
 	 */
-	public static function maybe_flush_rewrite_rules() {
```

Comment:
I realized this was unnecessary as it should have been merged into the existing `\AMP_Options_Manager::maybe_flush_rewrite_rules()`. This also ensures that the story rewrite rules actually get removed upon deactivation of the Stories experience.